dnf/plugins/repoquery/dnf-command-repoquery.c

```diff
@@ -85,6 +129,10 @@ dnf_command_repoquery_run (DnfCommand     *cmd,
   if (opt_available && opt_installed)
     opt_available = opt_installed = FALSE;
 
+  gboolean only_nevra = TRUE;
+  if (opt_info)
+    only_nevra = FALSE;
```

Comment:
`only_nevra = !opt_info`, I'd prefer to not introduce an auxiliary variable...

Reply:
Ok. I will remove the `only_nevra` variable. It is not needed now. (My previous version of repoquery supported other option and the variable was needed.)